BuildAndTest.cmd

```diff
@@ -5,52 +5,82 @@ SETLOCAL
 @REM create a nuget package for binskim) so must opt-in
 @REM %~dp0.nuget\NuGet.exe update -self
 
+set Configuration=%1
+set Platform=AnyCPU
+
+if "%Configuration%" EQU "" (
+set Configuration=Release
+)
+
+@REM Remove existing build data
+if exist bld (rd /s /q bld)
+
+SET NuGetConfigFile=%~dp0src\NuGet.config
+set NuGetPackageDir=.\src\packages

```

Comment:
Should do %~dp0 here, and down below as well

Reply:
Fixed across all folders.